.github/workflows/ci.yaml

```diff
@@ -8,7 +8,7 @@ jobs:
   build-and-test:
     uses: iExecBlockchainComputing/github-actions-workflows/.github/workflows/rust-build.yml@rust-build-v2.0.0
     with:
-      rust-version: '1.85.0'
+      rust-version: 'stable'
```

Comment:
Do we want `stable` or `1.88.0` here to keep an explicit version and decide when and how to check the update to a newer version ?